syntax = "proto3";

service SeatDrawer{
 rpc drawSpecificSeat(DrawSeatRequest) returns (DrewResultResponse) {}
 rpc drawAllSeats(stream DrawSeatRequest) returns (DrewResultResponse) {}
}

message DrawSeatRequest{
  int32 index = 1;
  int32 state = 2;
}

message DrewResultResponse{
 string result = 1;
}